2.7/Dockerfile

```diff
@@ -23,6 +23,17 @@ RUN yum install -y centos-release-scl && \
     rpm -e --nodeps centos-logos && \
     yum clean all -y
 
+# -- generated by https://github.com/rhcarvalho/envdiff, do not edit manually --
+# The ENV instruction below is equivalent to:
+#   scl enable python27
+ENV MANPATH=/opt/rh/python27/root/usr/share/man: \
+    X_SCLS=python27  \
+    LD_LIBRARY_PATH=/opt/rh/python27/root/usr/lib64 \
+    PATH=/opt/rh/python27/root/usr/bin:$PATH \
+    XDG_DATA_DIRS=/opt/rh/python27/root/usr/share \
+    PKG_CONFIG_PATH=/opt/rh/python27/root/usr/lib64/pkgconfig
```

Comment:
We might want to "fine-tune" certain _list_ variables to include a recursive reference to its previous value, even if it was previously undefined. Those wouldn't be possible to auto-detect.

For example:

```diff
-MANPATH=/opt/rh/python27/root/usr/share/man:
+MANPATH=/opt/rh/python27/root/usr/share/man:$MANPATH
# oddly, scl enable does leave a trailing ':' there!

-LD_LIBRARY_PATH=/opt/rh/python27/root/usr/lib64
+LD_LIBRARY_PATH=/opt/rh/python27/root/usr/lib64:$LD_LIBRARY_PATH

-XDG_DATA_DIRS=/opt/rh/python27/root/usr/share
+XDG_DATA_DIRS=/opt/rh/python27/root/usr/share:$XDG_DATA_DIRS

-PKG_CONFIG_PATH=/opt/rh/python27/root/usr/lib64/pkgconfig
+PKG_CONFIG_PATH=/opt/rh/python27/root/usr/lib64/pkgconfig:$PKG_CONFIG_PATH
```

The reason to do that is if we ever try to set a value for one of those variables before this opaque ENV instruction, or, harder to find out, a parent layer does set that variable, we'd be inadvertently overwriting the previous value.

To minimize human intervention, we can have a whitelist of variables that we want to give this special treatment, and have `envdiff` do the work for us.

@bparees @hhorak WDYT?